appveyor.yml

```diff
@@ -0,0 +1,14 @@
+init:
```

Comment:
Should we also set this in our appveyor files in the repos? We only set this in travis:

https://github.com/aspnet/DependencyInjection/blob/dev/appveyor.yml